src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,17 @@
+.stars {
+  display: flex;
+
+  &__star {
+    @include star-size($star-size);
+    margin-right: $star-margin-right;
+    background-image: url(../images/star-active.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+  }
+
+  &--4 {
+    .stars__star:nth-child(n+5) {
+      background-image: url(../images/star.svg);
+    }
+  }
+}
```

Comment:
Just a reminder to update the rest of star background modifiers from 1 to 3 and 5 for the grid catalog task x)